### ISSUE TITLE
Builder Doesn't Work with Symlinks

### ISSUE THREAD

**Issue by evshiron** [evshiron/nwjs-builder-phoenix#26]
```
{ Error: EISDIR: illegal operation on a directory, read
  cause: { Error: EISDIR: illegal operation on a directory, read errno: -21, code: 'EISDIR', syscall: 'read' },
  isOperational: true,
  errno: -21,
  code: 'EISDIR',
  syscall: 'read' }
```

No call stacks, when I was using `lerna` to link packages in the project.

**Comment by evshiron** [comment]
See jprichardson/node-fs-extra#191.

**Comment by TatriX** [comment]
Symlinks are broken now. They simply missing in the build dir.

**Comment by evshiron** [comment]
Fixed in `nwjs-builder-phoenix` 1.14.1, symlinks from project files will be dereferenced, to make things simpler and avoid issues on Windows.

**Comment by TatriX** [comment]
Thank you, I'll check it today.
